### ISSUE TITLE
rename renames the wrong thing (off by 1)

### ISSUE THREAD

**Issue by paulp** [mads-hartmann/ensime.tmbundle#8]
Next I tried out "rename" on a class.  It says "Rename AnyExt to what" in the dialogue so I know it knows what I highlighted: but when I kick off the rename, it instead renames the type parameter T on which AnyExt is parameterized (i.e. one identifier to the right.)

**Comment by paulp** [comment]
By the way, there's really no hurry on this stuff.  Focus on your real life.

**Comment by mads-hartmann** [comment]
Hey Paul,

I've been able to reproduce the error. Also, it works using Emacs so it's definitely my bundle that's broken. I'll look into it. 

On a side note. I just handed in my report which means my christmas vacation just started. It's the perfect time to fix bugs ;)

**Comment by mads-hartmann** [comment]
Okay, I've fixed the issue in the bundle. It will rename the right thing now, however, if you select the type parameter T it will rename the class now. This is a bug in Ensime as it also happens in Emacs. 

I've opened an issue: https://github.com/aemoncannon/ensime/issues/issue/114

I'm working on displaying the diff to the user before you accept the refactoring so the change isn't in master yet.
